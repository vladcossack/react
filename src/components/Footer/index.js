import React from 'react';
import { Footer } from '../styled.js';
import Logo from '../../assets/img/logo.svg';

const FooterBlock = () => (
  <Footer>
    <Logo />
  </Footer>
);

export default FooterBlock;