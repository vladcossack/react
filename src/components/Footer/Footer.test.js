import React from 'react';
import { render } from '@testing-library/react';

import Footer from '../Footer';

describe('Footer component', () => {
  it('should render Footer snapshot', () => {
    const { asFragment } = render(<Footer/>);
    expect(asFragment(<Footer/>)).toMatchSnapshot();
  })
}) 