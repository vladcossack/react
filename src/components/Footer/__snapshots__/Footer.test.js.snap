// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Footer snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-Axmtr JRjzr"
  >
    <svg
      data-file-name="SvgLogo"
    />
  </footer>
</DocumentFragment>
`;
